SparkPost/2web2ui
# SA-1747 - (UI) "is equal to" rendering within the ActiveFilters UI for some saved reports

[SA-1747](https://sparkpost.atlassian.net/browse/SA-1747)

### What Changed
- Adds additional check for filters in the initial state for the new comparator filters
- Adds new helper function that retrieves UI state for tags - distinct from form fields though using some of the same logic
- Renames report builder integration tests to match standard - they were a little more verbose than needed now that the feature has grown a good bit

### How To Test
- Verify tags still render as expected using the filters form and query params with the account flag `allow_report_filters_v2` set to `true`
- Verify that when report filters are in their initial state (`[{ AND: {} ]`) that the filters section is no longer rendered

### To Do
